ops/ansible/roles/postmaster_deploy/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
     - apache2
     - libapache2-mod-wsgi
     - libmysqlclient-dev
-  when: provision_type != "dev"
+  when: provision_type == "dev"
```

Comment:
Now this will only run when Vagrant calls it. I believe what we want is to install everything but MySQL for every provision type. Then for Vagrant, we want to install MySQL on top of that.